kie-pmml-bom/pom.xml

```diff
@@ -183,6 +183,22 @@
         <artifactId>kie-pmml-models-mining-evaluator</artifactId>
         <version>${version.org.kie}</version>
       </dependency>
+      <!-- Tree NO - CODEGEN -->
```

Comment:
What do you mean with `Tree NO - CODEGEN`?

Reply:
@danielezonca Fixed comment